scripts/setup_release_mirror.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "generating ssl certs"
+./usr/local/app/scripts/generate_ssl.sh
```

Comment:
nit: arguably could have just inlined that code here but it works either way

Reply:
this has been moved to the dockerfile and is inline